datasets/grillo-openeew.yaml

```diff
@@ -31,12 +31,12 @@ Resources:
 DataAtWork:
   Tutorials:
     - Title: Analyzing a magnitude 7.2 earthquake in Mexico using Python
-      URL: https://blog.grillo.io/analyzing-a-magnitude-7-2-earthquake-in-mexico-using-python-6272a4ff63e3
+      URL: https://openeew.com/docs/historic-data
```

Comment:
It looks like https://blog.grillo.io/analyzing-a-magnitude-7-2-earthquake-in-mexico-using-python-6272a4ff63e3 is still an active link, can that remain here? The link that it has changed to no longer goes to a tutorial, just docs.

Reply:
Thanks, yes we have been migrating everything to a dedicated OpenEEW documentation site, rather than a Grillo site (we are in the process of transitioning OpenEEW to become a Linux Foundation project, so need to separate the domains).

I realized i had the wrong link, it should have been https://openeew.com/docs/analyze-historic  - would it be ok If we changed to that ?

Reply:
Yep, thanks!